src/components/Todo/EditionForm.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import Datepicker from "react-datepicker";
+
+export default class EditionForm extends React.Component {
+  state = {
+    editTodoName: this.props.todo.name,
+    editDueDate: this.props.todo.dueDate
+   }
+
+  render() {
+    const { onUpdate, todo } = this.props;
+    const { editTodoName, editDueDate } = this.state;
+    const canSubmit = editTodoName === "" || editDueDate === null;
```

Comment:
There is an issue here. You `canSubmit` when both `editTodoName` and `editDueDate` are not null, so it is the contrary of what you have written !

Reply:
are you serious mdr i just copy/paste ta ligne de code dans ton commentaire précédent nan ?